backends/qualcomm/tests/test_qnn_delegate.py

```diff
@@ -231,6 +246,12 @@ def test_qnn_backend_mean_dim(self):
             with self.subTest(i=i):
                 self.lower_module_and_test_output(module, sample_input)
 
+    @unittest.skip("it will hang in runtime")
```

Comment:
Should you at least test quantization? And maybe separate quantization tests from runtime? I presume right now you cannot enable these in CI due to dependency on the device/libs

Reply:
I have test both of quantized and float-pointed operator.
https://github.com/CodeLinaro/executorch/blob/2dc87fe8352299e77701ab8e9019201f545a9a1b/backends/qualcomm/tests/test_qnn_delegate.py#L640